TTHAnalysis/python/tools/nanoAOD/susySOS_modules.py

```diff
@@ -205,23 +239,31 @@ def clean_and_FO_selection_SOS_noBtag(lep,year):
                                          or (abs(lep.pdgId)==13 and lep.softId ) )
 
 def fullCleaningLeptonSel(lep,year):
-	return clean_and_FO_selection_SOS(lep,year) and ( (abs(lep.pdgId)==11 and (VLooseFOEleID(lep, year) and lep.lostHits<=1))
+    return clean_and_FO_selection_SOS(lep,year) and ( (abs(lep.pdgId)==11 and (VLooseFOEleID(lep, year) and lep.lostHits<=1))
 										or (abs(lep.pdgId)==13 and lep.looseId) )
 def fullCleaningLeptonSel_noBtag(lep, year):
-	return clean_and_FO_selection_SOS_noBtag(lep,year) and ( (abs(lep.pdgId)==11 and (VLooseFOEleID(lep, year) and lep.lostHits<=1))
+    return clean_and_FO_selection_SOS_noBtag(lep,year) and ( (abs(lep.pdgId)==11 and (VLooseFOEleID(lep, year) and lep.lostHits<=1))
 										or (abs(lep.pdgId)==13 and lep.looseId) )
 
 def fullTightLeptonSel(lep,year):
-	return ( fullCleaningLeptonSel(lep,year) and ( abs(lep.pdgId)==13 or tightEleID(lep, year) )
+    return ( fullCleaningLeptonSel(lep,year) and ( abs(lep.pdgId)==13 or tightEleID(lep, year) )
 										and lep.pfRelIso03_all<0.5 and (lep.pfRelIso03_all*lep.pt)<5. and abs(lep.ip3d)<0.01 and lep.sip3d<2 )
 def fullTightLeptonSel_noBtag(lep,year):
-	return ( fullCleaningLeptonSel_noBtag(lep,year) and ( abs(lep.pdgId)==13 or tightEleID(lep, year) )
+    return ( fullCleaningLeptonSel_noBtag(lep,year) and ( abs(lep.pdgId)==13 or tightEleID(lep, year) )
 										and lep.pfRelIso03_all<0.5 and (lep.pfRelIso03_all*lep.pt)<5. and abs(lep.ip3d)<0.01 and lep.sip3d<2 )
 
 def tightLeptonSel_SOS(lep,year):
-	return ( clean_and_FO_selection_SOS(lep,year) and ( abs(lep.pdgId)==13 or tightEleID(lep, year) )
+    return ( clean_and_FO_selection_SOS(lep,year) and ( abs(lep.pdgId)==13 or tightEleID(lep, year) )
 										and lep.pfRelIso03_all<0.5 and (lep.pfRelIso03_all*lep.pt)<5. and abs(lep.ip3d)<0.01 and lep.sip3d<2 )
 
+def new_tightLeptonSel_SOS(lep,year):
```

Comment:
The tight ID did not change, therefore I don't see a reason why we should redefine it. It seems confusing and error-prone, since it is only a duplication.